packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -13,11 +13,14 @@
     />
     <ResizablePanels
       v-else
-      :offset-left="64"
-      :max-total-width="windowWidth"
+      class="w-[calc(100vw-64px)]"
```

Comment:
Can we interpolate the collapsedNavBarWidth value here?

Reply:
yes, and switching this to a plain old style attribute turned out to be the best way, we could sorta get Windi to do this by creating a CSS variable to pass to it, but the indirection gets very hard to keep up with at that point.